packages/chakra-ui-core/types/component.d.ts

```diff
@@ -0,0 +1,150 @@
+declare module '@chakra-ui/vue' {
+  import Vue from 'vue'
+
```

Comment:
Do you want to include declarations for the default export from `@chakra-ui/vue` also? Since the plugin has some options, I think it will be useful as well. You can find a good reference here:

https://github.com/chakra-ui/chakra-ui-vue/blob/e76b6572210848876821692ab7fe6c3fd8f2b8bd/packages/chakra-ui-core/src/Chakra/index.js#L28-L82

I think it can be improved since I last updated these.

Reply:
fix in 15ed202 and b7c9389

Reply:
Thank you @odanado